src/DependencyInjection/Configurator/GuzzleClientRequestIdConfigurator.php

```diff
@@ -34,6 +34,7 @@ public function __construct(GuzzleRequestIdInjector $guzzleRequestIdInjector)
 
     public function configure(Client $client)
     {
-        $client->getEmitter()->attach($this->guzzleRequestIdInjector);
+        $handlerStack = $client->getConfig('handler');
```

Comment:
This is the least impactful way of changing this. We need to configure [specific, configurable clients](https://github.com/OpenConext/Stepup-bundle/blob/e3c76e50cca7425c3f400b06c9b6445a489885bc/src/DependencyInjection/Configuration.php#L88-L94).